area_of_shape.cpp

```diff
@@ -0,0 +1,50 @@
+#include<iostream>
+using namespace std;
+
+
+  int main()
+   {
+    int side1,side2,option,flag=0;
+    float area=0;
+    while(!flag)
+    {
+     system("cls");
+     cout<<"\n********FIND THE AREA********\n\n";
+     cout<<"Enter the shape\n";
+     cout<<"1.Circle\n2.Square\n3.Rectangle\n4.Exit\n";
+     cout<<"Choice :";
+     cin>>option;
+
+     switch(option)
+      {
+       //circle
+       case 1:cout<<"Enter the radius of the circle in centimeters\n";
+	      cin>>side1;
+	      area=3.14*side1*side1;
+	      cout<<"The area of the circle = "<<area<<endl;
+	      system("pause");
```

Comment:
**pause** command would work only on windows machines.
you may replace _system("pause")_ with **getchar()**.